packages/palette/src/elements/Menu/Menu.tsx

```diff
@@ -14,20 +14,24 @@ import { Sans } from "../Typography"
 interface MenuProps {
   children?: React.ReactNode
   onClick?: (event: React.MouseEvent<HTMLElement>) => void
+  margin?: SpaceProps["m"]
+  padding?: SpaceProps["p"]
   title?: string
   width?: number | string
 }
 
 /** Menu */
 export const Menu: React.FC<MenuProps> = ({
   children,
+  margin = "2px",
+  padding = 1,
   title,
   width = 230,
   ...props
 }) => {
   return (
-    <MenuContainer width={width} m="2px" {...props}>
-      <BorderBox p={0} pb={1} background="white">
+    <MenuContainer width={width} m={margin} {...props}>
+      <BorderBox p={0} pb={padding} background="white">
```

Comment:
I think the `padding` name is a little confusing since, to a consumer of this component, I would expect it to affect the overall padding of the outermost component.

If this serves a specific purpose, could we rename? (`outerPadding` or `innerPadding` or something like that?)

Reply:
Yeah it is a bit confusing. I think I can refactor this a bit.